three-data-centers-in-two-cities-deployment.md

```diff
@@ -76,7 +76,7 @@ server_configs:
   tikv:
     server.grpc-compression-type: gzip
   pd:
-    replication.location-labels:  ["dc","zone","rack","host"]
+    replication.location-labels:  ["az","replication zone","rack","host"]
```

Comment:
waht is the replication zone?

Reply:
it's better use like `"region","zone","host"`.

Reply:
https://github.com/pingcap/docs-cn/blob/b4eff71b49db5fcaa5a6458cafec86f8f37ec48a/three-data-centers-in-two-cities-deployment.md?plain=1#L100-L117